packages/closer/components/custom-pages/CustomHero.tsx

```
@@ -53,20 +59,28 @@ const CustomHero: React.FC<{
             {content.title}
           </Heading>
-          <p
+
+          <div
             className={`${
               settings?.isInverted ? 'text-accent-light' : 'text-black'
             }`}
-          >
-            {content?.body}
-          </p>
+            dangerouslySetInnerHTML={{ __html: content.body }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)